tests/tsp.py

```diff
@@ -2,7 +2,7 @@
 
 from src import csv_read
 
-from src import tsp
+from src import Problem,Solver
 
 import numpy as np
 
```

Comment:
Add test with set random seed to to make sure that future versions do not alter the output of solve().
https://docs.python.org/3/library/random.html